tuned/daemon/daemon.py

```diff
@@ -228,6 +230,7 @@ def _thread_code(self):
 			i += 1
 
 		# if terminating due to profile switch
```

Comment:
I think this need a bit more verbose comment, because the if right bellow says the same thing and sets full_rollback. The naming is a bit confusing here.

Reply:
Comments added.